jobs/config.json

```diff
@@ -6887,6 +6887,7 @@
       "--gcp-project=kubernetes-scale",
       "--gcp-region=us-east1",
       "--gke-command-group=beta",
+      "--gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork name=ip-alias-subnet --cluster-ipv4-cidr=/12 --services-ipv4-cidr=/19",
```

Comment:
'ip-alias-subnet-regional'?
Should we use here the same name as in zonal cluster?

Reply:
Given that the networks should be different, I think this should be fine. Let me cross-check.